backend/cms/models/site.py

```diff
@@ -1,14 +1,35 @@
+from django.contrib.postgres.fields import ArrayField
 from django.db import models
 
+from cms.models.language import Language
+
 
 class Site(models.Model):
+    ACTIVE = 'acti'
+    HIDDEN = 'hidd'
+    ARCHIVED = 'arch'
+
     STATUS = (
-        ('publ', 'Public'),
-        ('priv', 'Private'),
-        ('arch', 'Archived'),
+        (ACTIVE, 'Active'),
+        (HIDDEN, 'Hidden'),
+        (ARCHIVED, 'Archived'),
     )
-    name = models.CharField(max_length=250)
-    slug = models.CharField(max_length=60)
+
+    title = models.CharField(max_length=200)
+    name = models.URLField(max_length=60, unique=True)
     status = models.CharField(max_length=4, choices=STATUS)
-    pub_date = models.DateTimeField(auto_now_add=True)
+    supported_languages = models.ManyToManyField(Language)
+
+    extras_enabled = models.BooleanField(default=True)
```

Comment:
in my opinion, `extras_enabled` should not be a class-attribute of sites, but should be derived from the extras of the specific site. in other words, the API should return `true` if at least one extra is activated and `false` otherwise.

Reply:
Yes, we can do that.
`events_enabled` is appropriate though, right?

Reply:
But we need to be able to determine which extras are activated. Am I getting something wrong?

Reply:
What about something like `extras_enabled = models.ManyToManyField(Extra)`?

Reply:
If we follow the proposal of this PR, where all the extras are hardcoded in the models, then it makes sense to use an 1:n relation. However, if we prefer the solution i suggested [here](https://github.com/Integreat/cms-django/pull/95#discussion_r244511383), we should go for the n:n approach. Otherwise we would store the same extra information (name, alias, url, etc...) multiple times for every single site and if e.g. every site had its own lehrstellenradar extra and the lehrstellenradar url changed, we would have to change it for every site individually.

Reply:
@timoludwig Correct. Just mentioning, that in this current PR not *all* extras would be hardcoded models, but just those that have a special behavior (with per-site necessary settings). All the other extras are realized with manageable SimpleLinkExtras.

Reply:
As stated above, I withdraw my proposal and agree on the 1:n relation of Site:Extra. So then we don't need the `extras_enabled` attribute, because we can just check if the `extra_set.all()` is empty, right?
`extras_enabled = Site.objects.get(id=site_id).extra_set.all() == []`